config/config.yaml

```diff
@@ -27,8 +27,22 @@ plank:
 branch-protection:
   allow_disabled_policies: true
   orgs:
+    # TODO: enforce dco check org-wide
+    # protect: true
+    # required_status_checks:
+    #   contexts:
+    #   - dco
```

Comment:
Can we not do this now already? I'd have thought `cert-manager` as an org should definitely have it enabled org wide (it wasn't with jetstack due to various internal projects/things that didn't use Prow)

Reply:
Done.

Reply:
Done...I think...I'm not actually sure where to put that stanza. After 3 attempts I've put it beneath the cert-manager org so that it only affects repos in that org. 
Lets see if the linter allows this.